studsapp/containers/settingsView/settingsView.js

```diff
@@ -3,28 +3,33 @@ import {
     StyleSheet, 
     Dimensions,
     View,
-    Text
+    Text,
+    Image
 } from 'react-native';
 import LinearGradient from 'react-native-linear-gradient';
 import Button from 'studsapp/generalComponents/button';
 import { removeData } from 'studsapp/utils/storage';
 
+const imageSource = 'studsapp/static/images/logo.png';
+
 class SettingsView extends React.Component {
     logOut = () => {
         removeData('token');
```

Comment:
Kanske flytta removeData anropet in i thunken som updaterar storen?

Reply:
Fixed